bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/ImageList.java

```diff
@@ -389,10 +396,10 @@ private void setForAllHandles(int index, Image image, int count) {
 	zoomToHandle.forEach((zoom, handle) -> set(index, image, count, handle, zoom));
 }
 
-void set (int index, Image image, int count, long handle, int zoom) {
+void set (int index, Image image, int count, long listHandle, int zoom) {
```

Comment:
Is this parameter renaming necessary/reasonable?

Reply:
It is shadowing the class attribute handle, so I would strongly argument for changing it. That is an easy source for a bug

Reply:
I see. I actually didn't want to challenge the change itself but understand how it is related to the PR's goal. If I understand correctly, it is a refactoring that could be done independently. But it's also fine for me to merge that simple change with this PR.